oslo/Util.js

```diff
@@ -1,5 +1,10 @@
 'use strict';
 
+const ForbiddenCharacterAllowedList = [
+	'd2l-activities\\activityEditor',
+	'@d2l\\d2l-attachment\\d2l-attachment'
+];
+const ForbiddenCharacters = ['.'];
 const EscapeRegExp = /([^a-zA-Z0-9\\:_-])/g;
```

Comment:
```suggestion
const EscapeRegExp = /([^a-zA-Z0-9\\:_-@])/g;
```
Do we want to add `@` to this?